contrib/buildrefactor/src/python/pants/contrib/buildrefactor/peek.py

```diff
@@ -0,0 +1,46 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+#./pants peek --path=src/scala/org/pantsbuild/zinc/analysis --target-name=zinc-analysis
```

Comment:
Remove these comments, or replace them with the valid syntax (`./pants peek --line_number=true src/scala/org/pantsbuild/zinc/analysis:zinc-analysis`)

(Probably easiest to just remove them; I think CI is failing because they're here too: https://travis-ci.org/pantsbuild/pants/jobs/308864341 )

Reply:
Removed the comments for CI. 

Is it relevant to include an example in the description? I noticed some Pants tasks have them but most do not.